prow/daily-e2e-cluster_wide-auth.sh

```diff
@@ -63,11 +63,4 @@ tar -xzf ${DAILY_BUILD}-linux.tar.gz
 cp -R ${DAILY_BUILD}/install/* install/
 
 echo 'Running Integration Tests'
-./tests/e2e.sh --auth_enable --cluster_wide "$@" \
-  --mixer_tag "${TAG}"\
-  --mixer_hub "${HUB}"\
-  --pilot_tag "${TAG}"\
-  --pilot_hub "${HUB}"\
-  --ca_tag "${TAG}"\
-  --ca_hub "${HUB}"\
```

Comment:
So does this mean that we are no longer testing the docker images from GCR?

Reply:
Yes we are 
https://github.com/istio/istio/blob/master/tests/istio.mk#L48